pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -60,55 +59,57 @@ Some simplifications require more than one step. For example:
 ]
 
 An expression that is not a @tech{value} can always be partitioned
-into two parts: a @deftech{redex}, which is the part that changed in a
+into two parts: a @deftech{redex} (``reducible expression''),
+which is the part that can change in a
 single-step simplification (highlighted), and the
 @deftech{continuation}, which is the evaluation
-context surrounding an expression. In @racket[(- 4 (+ 1 1))], the redex is @racket[(+ 1 1)], and
-the continuation is @racket[(- 4 @#,hole)], where @hole takes the
-place of the redex. That is, the continuation says how to ``continue''
+context surrounding the redex. In @racket[(- 4 (+ 1 1))], the redex is @racket[(+ 1 1)], and
+the continuation is @racket[(- 4 @#,hole)], where @hole indicates the
+position of the redex. That is, the continuation says how to ``continue''
 after the @tech{redex} is reduced to a @tech{value}.
 
-Before some things can be evaluated, some sub-expressions must be
-evaluated; for example, in the application @racket[(- 4 (+ 1 1))], the
+Before some expressions can be evaluated, their sub-expressions must be
+evaluated. For example, in the application @racket[(- 4 (+ 1 1))], the
 application of @racket[-] cannot be reduced until the sub-expression
 @racket[(+ 1 1)] is reduced.
-
 Thus, the specification of each syntactic form specifies how (some of)
-its sub-expressions are evaluated, and then how the results are
+its sub-expressions are evaluated and then how the results are
 combined to reduce the form away.
 
 The @deftech{dynamic extent} of an expression is the sequence of
-evaluation steps during which the expression contains the @tech{redex}.
+evaluation steps starting when the expression is selected as a @tech{redex}
+and continuing until it is fully evaluated.
 
 @;------------------------------------------------------------------------
 @section{Tail Position}
 
 An expression @racket[_expr1] is in @deftech{tail position} with
 respect to an enclosing expression @racket[_expr2] if, whenever
 @racket[_expr1] becomes a redex, its @tech{continuation} is the same
-as was the enclosing @racket[_expr2]'s @tech{continuation}.
+as the enclosing @racket[_expr2]'s @tech{continuation}.
 
 For example, the @racket[(+ 1 1)] expression is @italic{not} in @tech{tail
 position} with respect to @racket[(- 4 (+ 1 1))]. To illustrate, we use
 the notation @sub[_C _expr] to mean the expression that is produced by
-substituting @racket[_expr] in place of @hole in the @tech{continuation}
+substituting @racket[_expr] in place of @hole in some arbitrary @tech{continuation}
```

Comment:
Could you explain what you’re trying to clarify with this change? I don’t understand what is unclear about using “the”, but I could be missing something.

Reply:
I was trying to clarify that C is arbitrary with respect to the hole. Perhaps I ended up suggesting that the continuation might not be the one "surrounding" the hole.